Sell/Payment/Models/PaymentIntent/OrderType.swift

```diff
@@ -13,7 +13,7 @@ import GraphQL
 
 public enum OrderType: String, Codable {
 
-    case foodAndBeverage
+    case foodAndBeverage = "food_and_beverage"
```

Comment:
Why do we need to change it here? Assuming it's only required of `PaymentIntentInput`, it's converted to `apolloType` which holds the correct rawValue in `PaymentIntentInput.apolloInput.orderType`.

Reply:
It's because when making createMyPaymentIntent mutation, we need to convert BasketOrderType(model in Frontier app) to PaymentIntentInput. The paymentIntentOrderType is String `food_and_beverage`, so for the OrderType in sdk, it's rawValue is `foodAndBeverage`, it doesn't match and will return nil. 

`OrderType(rawValue: basket.orderType.paymentIntentOrderType)`

```
enum BasketOrderType {
    case foodAndBeverage(order: Order)
    case ticket(ticketProductBasket: TicketProductBasket)

    var paymentIntentOrderType: String {
        switch self {
        case .foodAndBeverage: return "food_and_beverage"
        case .ticket: return "ticket"
        }
    }
}
```

Reply:
Then it should be fixed on the platform, not the SDK in my opinion. The reason being that the data models' would be more clear and independent between SDK's model and Apollo model.

Reply:
I've removed it, thank you.